documentation/modules/auxiliary/scanner/ftp/anonymous.md

```diff
@@ -0,0 +1,93 @@
+## Description
+
+This module allows us to scan through a series of IP Addresses and provide details whether anonymous access is allowed or not. By default, anonymous access is not allowed by the FTP server.
```

Comment:
first sentence is missing the word FTP or FTP service, by itself it sounds like you can anonymously log in to any IP